.pre-commit-config.yaml

```
@@ -47,4 +46,11 @@ repos:
         language: system
         types: [python]
         pass_filenames: false # Run full test suite
-        stages: [pre-commit] # Only run on commit
+        stages: [pre-commit] # Only run on commit 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)